config/gregtech/worldgen/overworld/kaolinite_vein.json

```diff
@@ -1,35 +1,40 @@
 {
-  "weight": 60,
-  "density": 0.5,
-  "max_height": 70,
-  "min_height": 50,
-
-  "generator": {
-    "type": "ellipsoid",
-    "radius": [13, 13]
-  },
-  "filler": {
-    "type": "simple",
-    "value": {
-      "type": "weight_random",
-      "values": [
-        {
-          "weight": 30,
-          "value": "ore:kaolinite"
-        },
-        {
-          "weight": 30,
-          "value": "ore:zeolite"
-        },
-        {
-          "weight": 20,
-          "value": "ore:fullers_earth"
-        },
-        {
-          "weight": 20,
-          "value": "ore:glauconite_sand"
+    "weight": 60,
+    "density": 0.5,
+    "max_height": 70,
+    "min_height": 50,
+    "generator": {
+        "type": "ellipsoid",
+        "radius": [
+            13,
+            13
+        ]
+    },
+    "filler": {
+        "type": "simple",
+        "value": {
+            "type": "weight_random",
+            "values": [
+                {
+                    "weight": 30,
+                    "value": "ore:kaolinite"
+                },
+                {
+                    "weight": 30,
+                    "value": "ore:zeolite"
```

Comment:
Zeolite actually requires a large elctrolyzer to process. This vein is mostly useless tbh. Feel free to remove or keep it. The player will need zeolite dust access at some point for chemistry purposes.

Reply:
I kept the vein, but removed the stock manipulator recipe